README.md

```diff
@@ -68,3 +68,69 @@ This requires `podman` and `sscg` to be available on the host.
    ```
    make check
    ```
+
+## Running on Kubernetes
+
+The app service can also be deployed on Kubernetes, in particular the
+[cockpit-dev project on the staging instance](https://console-openshift-console.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/k8s/ns/cockpit-dev/pods).
+There is the [app service image BuildConfig](./webconsoleapp-k8s-buildconfig.yaml) and the [pods and services](./webconsoleapp-k8s.yaml).
+Both get deployed with
+
+    make k8s-deploy
+
+
+1. Validate that you can reach the deployment with:
```

Comment:
This step requires a VPN right?

Reply:
yes, for the staging instance. I'll add that.

Reply:
Added.